modules/simple_forms_api/app/services/simple_forms_api/s3_service/utils.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module SimpleFormsApi
+  module S3Service
+    class Utils
+      private
+
+      def assign_instance_variables(defaults)
+        defaults.each do |key, value|
+          instance_variable_set("@#{key}", value)
+        end
+      end
+
+      def log_info(message, **details)
+        Rails.logger.info(message, details)
+      end
+
+      def log_error(message, error, **details)
+        Rails.logger.error(message, details.merge(error: error.message, backtrace: error.backtrace.first(5)))
+      end
+
+      def handle_error(message, error, context)
+        raise error unless run_quiet
+
+        log_error(message, error, **context)
+        failures << { message:, error:, **context }
+      end
+
+      def s3_resource
+        @s3_resource ||= Reports::Uploader.new_s3_resource
```

Comment:
@kylesoskin @pennja It looks like we're using `Reports::Uploader` here (which relies on `Settings.reports.aws`) rather than something that relies on `Settings.form526_backup.aws` or `Settings.form526_export.aws`. Do we think it matters which secrets we use? I guess we're using `Reports::Uploader` because that is what `form526.rake` uses? Is that right?